hr_employee_nrq/views/hr_employee_views.xml

```diff
@@ -1,18 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-
     <record id="view_employee_form" model="ir.ui.view">
         <field name="name">hr.employee.form</field>
         <field name="model">hr.employee</field>
-        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="priority">99</field>
```

Comment:
Manually added priority here

Reply:
@TimLai125 Do you think it's more appropriate to make the address_home_id field invisible instead of replacing it?

Reply:
I tested with using invisible, but it seems the one of the field's value become empty when there are two fields (many2one, many2many).